### PR TITLE
fix(Global Search/Tagging): re-gen services with latest API definition

### PR DESCRIPTION
**Pull request by fabrizio-leoni**
## PR summary

New documentation about `replace` on get-tags swagger
## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by fabrizio-leoni** [comment]
image

**Comment by fabrizio-leoni** [comment]
image

File: .npmrc [hunk 1]
```diff
@@ -1 +1,2 @@
-registry=https://registry.npmjs.com/
\ No newline at end of file
+registry=https://registry.npmjs.com/
```
Comment (padamstx) [inline .npmrc:1]: Please revert your change to this file.  I don't know whether this is causing the build problem or not, but there should be no need to change this file.

**Review by padamstx — CHANGES_REQUESTED** [review]

File: .npmrc (hunk 1 above)
Comment (fabrizio-leoni) [reply .npmrc]: done

File: package.json [hunk 2]
```diff
@@ -59,7 +59,7 @@
     "eslint-plugin-node": "^9.0.0",
     "eslint-plugin-prettier": "^3.0.1",
     "installed-check": "^2.2.0",
-    "jest": "^29.5.0",
+    "jest": "^29.7.0",
```
Comment (padamstx) [inline package.json:62]: Please revert this change unless there is a good reason for making it.

**Review by padamstx — CHANGES_REQUESTED** [review]
Need to resolve why the change to package.json was made.

**Review by padamstx — APPROVED** [review]
PR now looks good.
